### DISCUSSION TITLE
No Docker image for 4.13?

### DISCUSSION THREAD

**Discussion by mig5** [WeblateOrg/weblate#7777] [Q&A] [answered]
Hi,

Weblate 4.13 was released on June 15 https://github.com/WeblateOrg/weblate/releases/tag/weblate-4.13

However, there is no Docker image with a 4.13 tag on Docker Hub, and the `:latest` tag dates from a month ago.

Can we expect a tagged Docker image for 4.13? The org I work with is building from specific version tags (we aren't using `:latest`) and we are keen to upgrade to the new release.

Thanks!

**Comment by mig5** [comment, marked as answer]
I see we finally got a new :latest and a [4.13-1](https://hub.docker.com/layers/weblate/weblate/weblate/4.13-1/images/sha256-5c64c11f948d126bef3f1443102d9daced960c7f6aca9c559fa4819eee0bdba5?context=explore) tag at Docker hub, a couple days after the release. Thanks

**Comment by nijel** [comment]
I indeed forgot to tag it, so it was delayed a bit.
